sources/mysql/data.go

```diff
@@ -64,8 +64,10 @@ func ConvertData(conv *internal.Conv, srcTable string, srcCols []string, srcSche
 		if vals[i] == "<nil>" || vals[i] == "NULL" {
 			continue
 		}
-		spColDef, ok1 := spSchema.ColDefs[spCol]
-		srcColDef, ok2 := srcSchema.ColDefs[srcCol]
+		tableId, _ := internal.GetTableIdFromName(conv, srcTable)
+		columnId, _ := internal.GetColumnIdFromName(conv, tableId, srcCol)
+		spColDef, ok1 := spSchema.ColDefs[columnId]
+		srcColDef, ok2 := srcSchema.ColDefs[columnId]
```

Comment:
This shouldn't be using reverse lookups using the `srcTable`, the method signature should get modified instead to receive `tableId` as input instead.
`tableId` should be passed in throughout the code, and the `tableName` should just be treated as one of the properties of table i.e it should not be used as an identifier.

Reply:
Updated to accept tableId as argument.